backend/api/v2beta1/pipeline.proto

```diff
@@ -173,7 +173,10 @@ message PipelineVersion {
   // one package file location.
   Url package_url = 6;
 
-  // Required input field. Specifies the pipeline spec for the pipeline version.
+  // Optional input field. Pipeline version code source URL.
```

Comment:
NIT: add a comment explaining differences between `package_url` and `code_source_url`.